lib/pxutil.js

```diff
@@ -205,6 +206,24 @@ function parseCookieHeader(headerValue) {
     return res;
 }
 
+function accessNestedObjectValueByStringPath(path, obj, separator='.') {
```

Comment:
It means that the customer should put the path to the credentials in a request as a string with dot separator?
I understand it's a parameter but the customer won't change the call to this function so it cannot really use a different separator, and if he can't who tells him that it must be provided with dot separator?

Reply:
maybe as part of the configuration he would also provide the separator symbol?

Reply:
The reason to have this as a parameter was mostly to make the function more generic in case we would want to use it for other purposes or change it in the future. I think having the separator as a config for the customer would be too much but that's just me.